Update Occupancy.R

jenzhao42 opened the pull request:
Completely revamped occupancy script, fingers crossed.

Commit:
Update Occupancy.R

Completely revamped occupancy script, fingers crossed.

jbylsma commented:
The original branch has been rebased on master; it didn't have the memory limit updates so SIANCT API was throwing 500s. We'll keep this PR open until everything is working.

Commit:
Update Occupancy.R

facedesking - adding read.csv

Commit:
Update Occupancy.R

dummy me

Commit:
Update Occupancy.R

sad face Deployment.ID should be Deploy.ID because this wasn't fixed in the occupancy line

Commit:
Update Occupancy.R

add in more frills

Commit:
Update Occupancy.R

switching position of x and y for pivot and cameras_dates

Commit:
Update Occupancy.R

Missing comma added

Commit:
Update Occupancy.R

swapping back pivot and cameras_dates because it looks better that way

Commit:
Update Occupancy.R

making the clump back to days instead of tons of seconds

Commit:
Update Occupancy.R

see if this fixes the deleting random column issue

Commit:
Change Relative Abundance

Change graph to go from most to least

Commit:
fix Occupancy Script

prepared for Deploy.ID being fixed to Deployment.ID some day
added warning message to csv rather than printing a warning message to get lost in the logs
fixed cast to dcast to account for new reshape2
check samplePeriod is good

Commit:
Update Occupancy.R

taking out a superfluous test code. the rest of this code block fixes the missing column issue from before
